pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -443,6 +445,14 @@ public TimestampUtils getTimestampUtils() {
    */
   protected Map<String, Class<?>> typemap = new HashMap<String, Class<?>>();
 
+  /**
+   * Obtain the connection lock and return it. Callers should use try-with-resources to ensure that
```

Comment:
Shouldn't this state '... must use try-with-resources...' ?

Reply:
I have updated the javadoc text here to use **_must_**